src/sensitivities/linalg/factorization/cholesky.jl

```diff
@@ -41,3 +41,19 @@ function ∇(
 )
     return getproperty(X̄, Symbol(uplo))
 end
+
+# Yar, some work arounds for breaking changes in ChainRules.jl
+# https://github.com/JuliaDiff/ChainRules.jl/pull/630
+
+# Single arg function was dropped
+function ChainRules.rrule(::typeof(cholesky), A::AbstractMatrix{<:Real})
+    return ChainRules.rrule(cholesky, A, Val(false))
+end
+
+# U and L properties were replaced with factors
+# This should probably be moved to ChainRules to support both options.
+function Base.getproperty(tangent::Tangent{P, T}, sym::Symbol) where {P <: Cholesky, T <: NamedTuple}
+    idx = hasfield(T, :factors) && sym in (:U, :L) ? :factors : sym
+    hasfield(T, idx) || return ZeroTangent()
+    return unthunk(getfield(ChainRulesCore.backing(tangent), idx))
```

Comment:
Shouldn't this return an upper/lower triangular matrix if `tangent.U`/`tangent.L` is requested? It seems this implementation would just return `tangent.factors` in both cases.

Reply:
IIRC in the CR PR it was also discussed if `getproperty` for these tangents should be added to CR.

Reply:
> Shouldn't this return an upper/lower triangular matrix if tangent.U/tangent.L is requested?

Yeah, this was just to see what was needed to make tests pass. AFAIK, `tangent.U` and `tangent.L` have just been renamed to `factors` shouldn't this just work as is in most cases? I guess the concern is that `factors` would be the incorrect type? 

Ideally, this is something that should be added to CR, but if folks disagree it can live here to keep things working.

Reply:
I think it's worth at least opening an issue on ChainRules to discuss adding this there.

Reply:
https://github.com/JuliaDiff/ChainRules.jl/issues/640